fem/libceed/convection.hpp

```diff
@@ -12,21 +12,29 @@
 #ifndef MFEM_LIBCEED_CONV_HPP
 #define MFEM_LIBCEED_CONV_HPP
 
-#include "ceed.hpp"
-
+#include "pa_integrator.hpp"
 #include "../fespace.hpp"
 
 namespace mfem
 {
 
-/// Initialize a Convection Integrator using libCEED
-void CeedPAConvectionAssemble(const FiniteElementSpace &fes,
-                              const mfem::IntegrationRule &ir,
-                              CeedData& ceedData);
-
-void CeedMFConvectionAssemble(const FiniteElementSpace &fes,
-                              const mfem::IntegrationRule &ir,
-                              CeedData& ceedData);
+class CeedPAConvectionIntegrator : public CeedPAIntegrator
```

Comment:
The name is pretty self-explanatory, but end users might use this object so a brief docstring would be helpful.